src/chains/models.py

```diff
@@ -32,6 +33,21 @@ def validate_native_currency_size(image: Union[str, IO[bytes]]) -> None:
         raise ValidationError("Image width and height need to be at most 512 pixels")
 
 
+def validate_tx_service_url(url: str) -> None:
+    result = urlparse(url)
+    if not all(
+        [
```

Comment:
Small tip: I think you can remove the `[]` comprehension here and use a generator `()`. That way if the first evaluates to `False` it will not continue. In this case is not relevant, though, as it's a really fast check

Reply:
Done in [d381e3b](https://github.com/gnosis/safe-config-service/pull/468/commits/d381e3ba966f02b54f8766ca862673c356f8cc20)